app/config/parameters.yml.dist

```diff
@@ -47,3 +47,5 @@ parameters:
     cas_authentication_version: 3
     cas_authentication_verify_ssl: true
     cas_authentication_certificate_path: null
+    enable_api_tracking: false
```

Comment:
So we can use this same parameter on the frontend lets call it just enable_tracking.